packages/web/src/builders/dev-server/schema.json

```diff
@@ -52,6 +52,16 @@
     "allowedHosts": {
       "type": "string",
       "description": "This option allows you to whitelist services that are allowed to access the dev server."
+    },
+    "memoryLimit": {
+      "type": "number",
+      "description": "Memory limit for type checking service process in MB.",
+      "default": 2048
```

Comment:
Since we are using  `ForkTsCheckerWebpackPlugin.DEFAULT_MEMORY_LIMIT,` I don't think we need `default` here

Reply:
Thanks, You're right.